cmd/fyne/internal/commands/build.go

```diff
@@ -282,6 +287,25 @@ func (b *Builder) build() error {
 	return err
 }
 
+func (b *Builder) computeSrcDir(fyneGoModRunner runner) (string, error) {
+	if b.goPackage == "" {
+		return b.srcdir, nil
+	}
+
+	srcdir := b.srcdir
+	if strings.HasPrefix(b.goPackage, "./") {
+		srcdir = filepath.Join(srcdir, b.goPackage)
+	} else {
+		// get the output of go env GOROOT
+		goroot, err := fyneGoModRunner.runOutput("env", "GOROOT")
```

Comment:
Why GOROOT? Isn't code stored in GOPATH?

Reply:
go build something is looking up in GOROOT for me.

Reply:
GOROOT is where the runtime and standard library are stored, not where user code goes